src/Sylius/Bundle/CoreBundle/Resources/translations/validators.en.yml

```
@@ -72,6 +72,7 @@ sylius:
         payment_method_eligibility: 'This payment method %paymentMethodName% has been disabled. Please reselect your payment method.'
         product_eligibility: 'This product %productName% has been disabled.'
         shipping_method_eligibility: 'Product does not fit requirements for %shippingMethodName% shipping method. Please reselect your shipping method.'
+        shipping_method_not_available: 'The "%shippingMethodName%" shipping method is not available, please choose another one.'
```

```suggestion
        shipping_method_not_available: 'The "%shippingMethodName%" shipping method is not available. Please reselect your shipping method.'
```
Same phrasing as above.